mops/src/opsa/opsa.cu

```diff
@@ -1 +1,393 @@
-// todo: cuda device code
+
```

Comment:
can we include this file (and all other .cu/.cuh files) in the files to be formatted with clang-format? @frostedoyster

Reply:
Ping again @frostedoyster 😃